daweb/js2/architektura/patch-delete.md

```diff
@@ -0,0 +1,3 @@
+## Dotaz PATCH
+
+Kromě dotazu GET a POST existují ještě dotazy PATCH a DELETE, který slouží k aktualizaci nějakého záznamu.
```

Comment:
> Kromě dotazu GET a POST existují ještě dotazy PATCH a DELETE, který slouží k aktualizaci nějakého záznamu.

Napsal bych:
Kromě metod požadavků `GET` a `POST` existují ještě metody `PUT` a `DELETE`. `PUT` slouží k aktualizaci nějakého záznamu, `DELETE` záznam smaže.

`PATCH` bych úplně vynechal, nekomplikoval byl to rozdílem mezi `PUT` a `PATCH`.